src/components/ItemsList.jsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+import { Loader } from "./Loader";
+
+const BASE_URL = "http://localhost:9000";
+
+function ItemsList() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [itemsList, setItemsList] = useState([]);
```

Comment:
this state stores data, and it should be reflected in its naming, eg:  [citiesData, setCitiesData]

Reply:
done